docs/exposing-eventlisteners.md

```diff
@@ -44,12 +45,12 @@ metadata:
     nginx.ingress.kubernetes.io/ssl-redirect: "false"
 spec:
   rules:
-  - http:
-      paths:
-      - path: /
-        backend:
-          serviceName: getting-started-listener-b8rqz # REPLACE WITH YOUR SERVICE NAME FROM STEP 2
-          servicePort: 8080
+    - http:
+        paths:
+          - path: /
+            backend:
+              serviceName: getting-started-listener-b8rqz # REPLACE WITH YOUR SERVICE NAME FROM STEP 2
+              servicePort: 8080
```

Comment:
@dibyom This is different from how we do the indentation in some of our YAML files (example [triggertemplate.yaml](https://github.com/tektoncd/triggers/blob/master/examples/triggertemplates/triggertemplate.yaml#L7)):
```yaml
...
spec:
  params:
  - name: gitrevision
    description: The git revision
    default: master
...
```

Why isn't Prettier updating these YAML files as well?

Reply:
good question :thinking: